src/formio/validators/plugins.js

```diff
@@ -0,0 +1,71 @@
+import {post} from "../../api";
+
+const {REACT_APP_BASE_API_URL} = process.env;
```

Comment:
This is not available runtime, only at build time. It should be taken from the `sdk.OpenForm.baseUrl` configuration. It's included in a React context, so we'll have to pass it down as a custom option _somehow_ to the Form/components.